addons/test/autoCorrectedMmddyyyyPipe.spec.js

```diff
@@ -28,8 +28,4 @@ describe('autoCorrectedMmddyyyyPipe', () => {
   it('returns false if day 1st digit is 0 and 2nd digit is also 0', () => {
     expect(autoCorrectedMmddyyyyPipe('12/00')).to.equal(false)
   })
-
-  it('completes the year to `200` if the 1st digit is 0', () => {
```

Comment:
Alright, I'm okay with removing it too then. I can push some more commits to this PR to rename the files and update the documentation.